src/CoreBundle/private/library/classes/TCMSFields/TCMSField.class.php

```diff
@@ -846,12 +846,12 @@ public function AllowRenameRelatedTablesBeforeFieldSave($aNewFieldData, $aOldFie
     /**
      * Renames existing related table.
      *
-     * @param string $sNewTableName
-     * @param bool   $returnDDL
+     * @param array $newFieldData
+     * @param bool  $returnDDL
      *
      * @return string|null
      */
-    public function RenameRelatedTables($sNewTableName, $returnDDL = false)
+    public function RenameRelatedTables($newFieldData, $returnDDL = false)
```

Comment:
Although this is more a fix than a change, we should a short note in the UPGRADE guide.